src/components/header.tsx

```diff
@@ -26,16 +63,33 @@ export const Header = () => {
       <Accordion onClick={() => {
         setIsExpanded(!isExpanded);
       }}>
-        <h4>Dam Assessment and Monitoring</h4>
+        <h4><u>Dam Assessment and Modelling</u></h4>
```

Comment:
The project's name is 'Dam Assessment and Monitoring':
```suggestion
        <h4><u>Dam Assessment and Monitoring</u></h4>
```